pkg/volume/util/fs/fs_windows_test.go

```diff
@@ -74,8 +75,7 @@ func TestDiskUsage(t *testing.T) {
 	if err != nil {
 		t.Fatalf("TestDiskUsage failed: %s", err.Error())
 	}
-	if size.Cmp(used) != 1 {
-		t.Fatalf("TestDiskUsage failed: %s", err.Error())
+	if size.Cmp(used) != 0 {
+		t.Fatalf("TestDiskUsage failed: expected 1, got %d", size.Cmp(used))
```

Comment:
mismatch:  expected 1   and size.Cmp(used) != 0

Reply:
updated thanks!